script/ci_runner.sh

```diff
@@ -53,8 +53,7 @@ run() {
   eval $3;
 }
 
-if [ $2 != '' ] && [ $3 != '' ]
-then
+if [ $2 != '' ] && [ $3 != '' ]; then
```

Comment:
If this triggers warning maybe a better way to check those vars are not empty would be the following one:

```bash
if [ -n "$2" ] && [ -n "$3" ]; then
```